src/main/java/org/jvnet/hudson/update_center/Plugin.java

```diff
@@ -254,22 +254,9 @@ public String getScmHost() {
         return labels;
     }
 
-    /** @return {@code true} if the plugin's wiki page has the "plugin-deprecated" label. */
-    public boolean isDeprecated() {
```

Comment:
IMHO the Jenkins project doesn't support custom update centers (especially closed/proprietary ones) and that isn't something we should worry about too much here.